migrations/deployed_addresses.md

```diff
@@ -19,6 +19,7 @@
 - PayableMarginMinter:             [0x059550a1ca3c46a2adb803e9e3ea4585a34f004a](https://etherscan.io/address/0x059550a1ca3c46a2adb803e9e3ea4585a34f004a)
 - BucketLenderFactory:             [0xfbc08a66f8bac391f21472d9625c1092ff41cc04](https://etherscan.io/address/0xfbc08a66f8bac391f21472d9625c1092ff41cc04)
 - EthWrapperForBucketLender:       [0x61fa58d7ef40d6f6853740330b62ccad34f8bc04](https://etherscan.io/address/0x61fa58d7ef40d6f6853740330b62ccad34f8bc04)
+- BucketLenderProxy:               N/A
```

Comment:
You don't just wanna deploy it to production now too?

Reply:
I was looking into doing it without doing the weird deploy thing... cause truffle sends a bunch of transactions to update the migrations and whatnot